client/fire-webclient/src/main/kotlin/uk/ac/ebi/fire/client/model/FireApiFile.kt

```diff
@@ -10,7 +10,7 @@ data class FireApiFile(
     val createTime: String,
     val filesystemEntry: FileSystemEntry? = null,
 ) {
-    val path: String? = filesystemEntry?.path
+    val path: String? = filesystemEntry?.path?.removePrefix("/")
```

Comment:
I think is worth it to document what we do this at class level or property level.